tests/tests/cpld_dcp_gefs

```diff
@@ -17,3 +17,7 @@ export CICE_NPROC=$ICE_tasks
 export np2=`expr $CICE_NPROC / 2`
 export CICE_BLCKX=`expr $NX_GLB / $np2`
 export CICE_BLCKY=`expr $NY_GLB / 2`
+
+if [[ ${MACHINE_ID} = ursa ]]; then
+   WLCLK=60
+if
```

Comment:
@jkbk2004 @BrianCurtis-NOAA This is incorrect and the RTs fail on Hera. Should be `fi`, no `if`